src/inet/tests/TestInetCommon.cpp

```diff
@@ -290,8 +290,8 @@ void InitNetwork()
         {
             uint64_t iid    = gNetworkOptions.LocalIPv6Addr[j].InterfaceId();
             char * tap_name = (char *) chip::Platform::MemoryAlloc(sizeof(gDefaultTapDeviceName));
+            assert(tap_name);
```

Comment:
This seems unrelated to this check-in.

Reply:
Split to https://github.com/project-chip/connectedhomeip/pull/3230